ogm/releases/5.3/index.adoc

```diff
@@ -0,0 +1,18 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.3"
+
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
+
+==== Hibernate Search 5.9 support
+
+As of Hibernate Search 5.9, the version of Search to use with Search is 5.9 (use the latest micro version).
+
```

Comment:
I wouldn't use the sub-title in this core section. Let's keep it consistent with the other releases and section.
I would write the the main purpose of this release is to work with ORM 5.2 and Search 5.9.